### PR TITLE
Add top-level readme

### PR DESCRIPTION
**Pull request by garbados**
This PR adds a top-level readme with basic information for new committers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+See the README.md of each respective project for further details, including
+installation, usage, and testing instructions.
+
+## Development
+
+Notes and guidelines for committers:
+
+- [Use tpope commit norms](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html): Capitalize commit messages, use the imperative present tense, wrap commit messages at 50 characters.
+- Check [Linear](https://linear.app/) for project backlogs, issues, discussion, etc.
```
Comment (xla) [inline README.md]: Are Linear backlogs public? If not it's a bit miss-leading to point to it in a public repo.

**Comment by rudolfs** [comment]
Now that the `radicle-upstream` repository represents the actual desktop application that people will use to interact with the radicle protocol, should we maybe make whatever is in the `app` directory top-level and have the proxy as a sub-directory perhaps?

**Comment by xla** [comment]
@rudolfs I still don't agree with that strategy. If we want to correct this, a hierarchy could look like this:

```
radicle-upstream/
|-- proxy/
     |-- src/
     |-- Cargo.toml
     ...
|-- ui
     |-- src/
     |-- package.json
     |-- rollup.config.js
     ...
|-- READM.md
|-- main.js
|-- rollup.config.js
```

Where the top-level is the app container (electron).

**Comment by rudolfs** [comment]
@xla: ok, never mind then. Top-level README.md it is.

File: README.md (hunk 1 above)
Comment (garbados) [reply README.md]: Oh, this repo is public isn't it. Yes that's true. I'll remove it.

**Comment by garbados** [comment]
Removed the linear link. Merging...

**Pull request merged**